Extensions/glowing_follow.js

```diff
@@ -14,10 +14,10 @@ XKit.extensions.glowing_follow = new Object({
 		this.running = true;
 		XKit.tools.init_css("glowing_follow");
 
-		if ($(".follow").length > 0) {
+		var follow_button = $(".follow-button");
```

Comment:
Would prefer this to be a new XKit.iframe function. (because I think some blogs still might not have the new controls)